web/cobrands/sass/_base.scss

```diff
@@ -1647,6 +1647,10 @@ input.final-submit {
     left: 0px;
 }
 
+#fancybox-bg-s {
```

Comment:
Might as well be `.fancybox-bg` to treat all the shadows the same?

Reply:
Thanks Matthew, I just pushed a commit that will apply the the fix in all the ".fancybox-bg" elements instead of just the bottom one.